guides/v2.3/graphql/release-notes.md

```diff
@@ -0,0 +1,43 @@
+---
+group: graphql
+title: Release Notes
+---
+
+*Release notes published March 2019.*
+
+GraphQL is a flexible and performant API that allows you to build custom front-ends, including headless storefronts, [Progressive Web Apps](https://github.com/magento-research/pwa-studio) (PWA), and mobile apps for Magento.
+
+The **[Magento GraphQL](https://github.com/magento/graphql-ce) project** is a Magento Community Engineering special project open to contributors.
+To take part and contribute, see the [Magento GraphQL](https://github.com/magento/graphql-ce) repository and [wiki](https://github.com/magento/graphql-ce/wiki) to get started. Join us in our [Slack](https://magentocommeng.slack.com/messages/C8076E0KS) channel (or [self signup](https://tinyurl.com/engcom-slack)) to discuss the project.
+
+These release notes include:
+
+-   {:.new}New features
+-   {:.fix}Fixes and improvements
```

Comment:
are the fixes coming later? I didn't see referenced below.

Reply:
Lori advised to keep the Fixes line. Changed the lead-in sentence to "...can include:"